lms/djangoapps/lti_provider/views.py

```diff
@@ -154,13 +159,12 @@ def render_courseware(request, lti_params):
     :return: an HttpResponse object that contains the template and necessary
     context to render the courseware.
     """
-    usage_id = lti_params['usage_id']
-    course_id = lti_params['course_id']
-    course_key = CourseKey.from_string(course_id)
+    usage_key = lti_params['usage_key']
+    course_key = lti_params['course_key']
     user = request.user
     course = get_course_with_access(user, 'load', course_key)
     staff = has_access(request.user, 'staff', course)
-    instance, _ = get_module_by_usage_id(request, course_id, usage_id)
+    instance, _ = get_module_by_usage_id(request, str(course_key), str(usage_key))
```

Comment:
Please don't use `_` as a placeholder variable. It overloads the meaning of `_ = ugettext`, a common shorthand (across many code systems) where underscore is aliased to the gettext (translation) function.

For placeholder variables, please use either `__` or `_dummy` (I prefer the latter, it's a bit clearer)